src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import cn from 'classnames';
+
+export const PeoplePage: FC = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const { personSlug } = useParams();
+
+  const personInfo = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+  const personName = (name: string | null) => {
+    return peoples.find(person => person.name === name);
+  };
+
+  useEffect(() => {
+    setLoading(true);
+
+    getPeople()
+      .then(setPeoples)
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
```

Comment:
The `useEffect` hook is used correctly to fetch data. However, it's recommended to handle the state update in the `.then` block instead of passing `setPeoples` directly to it. This would allow you to handle the response data before setting the state, which could be useful if the data needs to be transformed or if additional error handling is required.